hack/installer/entrypoint.sh

```diff
@@ -16,15 +16,25 @@ EOF
 
 function setup_raw_device(){
   printf "Creating RAW device, this may take a moment..."
+
   if [[ -f ${TALOS_RAW} ]]; then
     rm ${TALOS_RAW}
   fi
+
+  # If defaults or a test file is specified, make a temp file
+  # to prevent collisions.
+  if [ ${TALOS_RAW} == "/out/talos.raw" ] || \
+     [ ${TALOS_RAW} == "/out/test.raw" ]; then
+    TALOS_RAW="$(mktemp ${TALOS_RAW}.XXXXXX)"
+    RENAME_TEMPFILE=true
+  fi
```

Comment:
Still not sure how I feel about the above.

Reply:
Agree.